bin/heudiconv

```diff
@@ -1,2160 +0,0 @@
-
-            # might have been made R/O already
-            os.chmod(json_phasediffname, 0o0664)
-            json.dump(json_, open(json_phasediffname, 'w'), indent=2)
-            os.chmod(json_phasediffname, 0o0444)
-
-        # phasediff one should contain two PhaseDiff's
-        #  -- one for original amplitude and the other already replicating what is there
-        # so let's load json files for magnitudes and
-        # place them into phasediff
-
-
-def embed_metadata_from_dicoms(converter, is_bids, item_dicoms, outname,
-                               outname_bids, prov_file, scaninfo, tmpdir,
-                               with_prov, min_meta):
-    """
-    Enhance sidecar information file with more information from DICOMs
-
-    Parameters
-    ----------
-    converter
-    is_bids
-    item_dicoms
-    outname
-    outname_bids
-    prov_file
-    scaninfo
-    tmpdir
-    with_prov
-    min_meta
-
-    Returns
-    -------
-
-    """
-    from nipype import Node, Function
-    embedfunc = Node(Function(input_names=['dcmfiles',
-                                           'niftifile',
-                                           'infofile',
-                                           'bids_info',
-                                           'force',
-                                           'min_meta'],
-                              output_names=['outfile',
-                                            'meta'],
-                              function=embed_nifti),
-                     name='embedder')
-    embedfunc.inputs.dcmfiles = item_dicoms
-    embedfunc.inputs.niftifile = os.path.abspath(outname)
-    embedfunc.inputs.infofile = os.path.abspath(scaninfo)
-    embedfunc.inputs.min_meta = min_meta
-    if is_bids and (converter == 'dcm2niix'):
-        embedfunc.inputs.bids_info = load_json(os.path.abspath(outname_bids))
-    else:
-        embedfunc.inputs.bids_info = None
-    embedfunc.inputs.force = True
-    embedfunc.base_dir = tmpdir
-    cwd = os.getcwd()
-    try:
-        """
-        Ran into
-INFO: Executing node embedder in dir: /tmp/heudiconvdcm2W3UQ7/embedder
-ERROR: Embedding failed: [Errno 13] Permission denied: '/inbox/BIDS/tmp/test2-jessie/Wheatley/Beau/1007_personality/sub-sid000138/fmap/sub-sid000138_3mm_run-01_phasediff.json'
-while
-HEUDICONV_LOGLEVEL=WARNING time bin/heudiconv -f heuristics/dbic_bids.py -c dcm2niix -o /inbox/BIDS/tmp/test2-jessie --bids --datalad /inbox/DICOM/2017/01/28/A000203
-
-so it seems that there is a filename collision so it tries to save into the same file name
-and there was a screw up for that A
-
-/mnt/btrfs/dbic/inbox/DICOM/2017/01/28/A000203
-        StudySessionInfo(locator='Wheatley/Beau/1007_personality', session=None, subject='sid000138') 16 sequences
-        StudySessionInfo(locator='Wheatley/Beau/1007_personality', session=None, subject='a000203') 2 sequences
-
-
-in that one though
-        """
```

Comment:
I have finally caught this issue and fixed for it in #105 . Sorry for causing more conflicts/more work